src/test/java/lotto/domain/LottoNumbersTest.java

```diff
@@ -0,0 +1,91 @@
+package lotto.domain;
+
+import lotto.domain.exception.NotValidLottoNumberException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LottoNumbersTest {
+
+    @DisplayName("LottoNumbers 인스턴스 생성 테스트")
+    @Test
+    public void createLottoNumberInstance() {
+        //Given & When
+        LottoNumbers lottoNumbers = new LottoNumbers((capacity) -> new TreeSet<>(Arrays.asList(1, 3, 5, 6, 7, 8)));
+
+        //Then
+        assertThat(lottoNumbers).isNotNull();
+    }
+
+    @DisplayName("유효하지 않는 로또 사이즈일 경우 예외처리")
+    @ParameterizedTest
+    @MethodSource("createNotValidLottoNumber")
+    public void notValidLottoNumbersException(Set<Integer> lottoSet) {
+        assertThatThrownBy(() -> {
+            LottoNumbers lottoNumbers = new LottoNumbers((capacity) -> new TreeSet<>(lottoSet));
+        }).isInstanceOf(NotValidLottoNumberException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new LottoNumbers((capacity) -> new TreeSet<>(lottoSet)))
                       .isInstanceOf(NotValidLottoNumberException.class);
```
사용하지 않는 변수는 제거해주고 expression lambda를 사용해보면 어떨까요? 😄